psi4/src/psi4/libsapt_solver/amplitudes.cc

```diff
@@ -1132,7 +1132,7 @@ void SAPT2p3::ind30_amps(int AAfile, const char *ARlabel, int BBfile, const char
 
     free(X);
 
-    if (amplabel == "Ind30 uAR Amplitudes") {
+    if (strcmp(amplabel, "Ind30 uAR Amplitudes")) {
```

Comment:
I don't think this is correct - IIRC strcmp returns 0 if both strings are equal
```suggestion
    if (amplabel == std::string("Ind30 uAR Amplitudes")) {
```

Reply:
Is the std::string even necessary? I think the conversion happens automatically

Reply:
Nevermind...

Reply:
You were correct, Andy.